site/docs/pages/identity/get-name.mdx

```diff
@@ -46,6 +46,29 @@ const name = await getName({ address });
 
 :::
 
+Get name from another chain:
```

Comment:
Don't worry about the docs for now. We can add it later after we release this.

Otherwise as we merge the PR than the docs show something the code hasn't yet.